src/routes.js

```diff
@@ -0,0 +1,14 @@
+const usersControllers = require('./controllers/users')
```

Comment:
I think singular for these names.

In the same way the model is called User, not Users,

Call this userController